source-code-git/fs/test/__snapshots__/suite.test.ts.snap

```diff
@@ -0,0 +1,16 @@
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
+
+exports[`memoryFs 1`] = `
+{
+  "file2": "text in the second file",
+  "home": {
+    "user1": {
+      "documents": {
+        "file1": "text in the first file",
+      },
+      "downloads": {},
+      "file3": "text in the third file",
+    },
+  },
+}
```

Comment:
Can you add vitest snapshots to .gitignore?

Reply:
done